src/modules/pm.c

```diff
@@ -142,27 +162,74 @@ static void publish_susp_req(const bool new) {
     M_PUB(suspend_req);
 }
 
+static bool acquire_systemd_lock(void) {
+    inh_api = SYSTEMD;
+    SYSBUS_ARG_REPLY(pm_args, parse_bus_reply, &pm_inh_token,
+                     "org.freedesktop.login1", "/org/freedesktop/login1",
+                     "org.freedesktop.login1.Manager", "Inhibit");
+    int ret = call(&pm_args, "ssss", "idle", "Clight", "Idle inhibitor.", "block");
```

Comment:
@tpeacock19 i just noticed we are inhibiting `idle` here; i think we should be inhibiting `shutdown, sleep` instead.

At the same time, `idle` should be inhibited by the inhibit module IMHO, like here: https://github.com/FedeDP/Clight/tree/master/src/modules#L60
What do you think?

Reply:
I can do a PR in that direction myself btw ;) i want to just gather your feedback since you implemented it!

Reply:
Sorry, i meant here: https://github.com/FedeDP/Clight/blob/master/src/modules/display.c#L95

Reply:
> Seven distinct inhibitor lock types may be taken, or a combination of them:
> 
>   1. sleep inhibits system suspend and hibernation requested by (unprivileged) **users**
>   2. shutdown inhibits high-level system power-off and reboot requested by (unprivileged) **users**
>   3. idle inhibits that the system goes into idle mode, possibly resulting in **automatic** system suspend or shutdown depending on configuration.

From the [Freedesktop Docs](https://www.freedesktop.org/wiki/Software/systemd/inhibit/). I had assumed that idle would cover the cases for automatic suspend/sleep/shutdown and if it was explicitly called, then it should allow for it. That is how I understand the documentation anyway. Do you think differently?

Reply:
I think you are right indeed!
Thank you!